### PR TITLE
[terra-menu] Set showHeader to false, display header for bounded

### PR DESCRIPTION
**Pull request by sugan2416**
### Summary

**What was changed:**

- Updated `showHeader` prop default value to false
- Allow header title to display when menu goes bounded

**Why it was changed:**

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-XXXX 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-menu/src/_MenuContent.jsx [hunk 1]
```diff
@@ -346,6 +346,14 @@ class MenuContent extends React.Component {
     let index = -1;
     const totalItems = MenuUtils.totalItems(this.props.children);
     const items = this.props.children ? [] : undefined;
+    const boundingFrame = this.props.boundingRef ? this.props.boundingRef() : undefined;
+    const isFullScreen = MenuUtils.isFullScreen(
```
Comment (ShettyAkarsh) [inline packages/terra-menu/src/_MenuContent.jsx:350]: is this checking only for bounded menu ?

**Comment by ShettyAkarsh** [comment]
@sugan2416  will there be new scrensshots generated for bounded menu users ? as this changes will show the headers always going ahead for bounded menu ?

File: packages/terra-framework-docs/CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,9 @@
 * Added
   * Added documentation and tests for `terra-worklist-data-grid`.
 
+* Changed
+  * Updated example for `terra-menu`
```
Comment (sdadn) [inline packages/terra-framework-docs/CHANGELOG.md:18]:
```suggestion
  * Updated example for `terra-menu`.
```

**Review by sdadn — CHANGES_REQUESTED** [review]
Can this be rebased with main so that it is up to date?

**Comment by supreethmr** [comment]
Closing this PR as the branch has become stale and PR does not have enough details about the issue OR Jira it's addressing. I would suggest to create new PR if required 
cc : @sugan2416

**Pull request closed without merging**